database/postgres/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}
+
+VERSION="1.24.0"
+NAMESPACE="postgres"
+
+# Install cloudnative-pg operator.
+kubectl apply --server-side -f \
+  https://raw.githubusercontent.com/cloudnative-pg/cloudnative-pg/main/releases/cnpg-${VERSION}.yaml
+kubectl rollout status deployment --namespace=cnpg-system --timeout=90s
+
+# Create the postgres database server.
+kubectl create ns ${NAMESPACE}
+kubectl -n ${NAMESPACE} apply -f .
+kubectl -n ${NAMESPACE} wait pod/kubearchive-1 --for=create --timeout=60s
```

Comment:
This yields the following error:

```
+ kubectl -n postgres wait pod/kubearchive-1 --for=create --timeout=60s
error: unrecognized condition: "create"
```

Looks it should be `--for=condition=ready`. However that may not work if the command is issued before the Pod starts creating. Maybe you can wait for the `Cluster` object to be ready instead.

Reply:
No. I added a comment in the "Notes for Reviewers" section:
>It requires kubectl 1.31 or later.